### PR TITLE
Plugins rewrite

### PR DESCRIPTION
**Pull request by PeterKottas**

### CODE REVIEW

**Comment by PeterKottas** [comment]
Sometimes I fell like screaming into a pillow with git. That tool was not made for me ... But it's done, seems like all changes are there. Obviously all info from #630 applies.

**Comment by aeneasr** [comment]
You can always use git diff to create a diff patch off master - especially if no other changes have been merged this can help a ton. Then you check out master and apply the patch, this will guarantee all changed having moved over. This is similar (but way cleaner / faster) to copying all files out of the git dir, checking out a new branch off master, then copying the files back.

File: README.md [hunk 1]
```diff
@@ -149,13 +149,6 @@ IDE: vscode
 Vscode extensions: prettier, tslint, code spell checker, beautify css/sass/scss/less
 
 ## Known issues
-### Unlink error
```
[the remark this answers is not in the document]
Comment (PeterKottas) [reply README.md]: Yep, that's fixed :)

**Comment by PeterKottas** [comment]
Weird, I'll look into it.

**Comment by aeneasr** [comment]
Thanks!

**Comment by aeneasr** [comment]
One last thing, could you please sign your commit? :)

**Comment by PeterKottas** [comment]
Is this git fucking kidding me? :D Did exactly what it asked for, not it has conflicts ...

**Comment by PeterKottas** [comment]
Dude what the hell should I do with this? Why is everything a conflict all the sudden? Because I force pushed? Like ... it's the same code... why git ... why

**Pull request closed without merging**